docs/INSTALLATION.md

```diff
@@ -52,3 +53,26 @@ There are also support for the D language in various IDEs e.g.
 [D plugin](https://plugins.jetbrains.com/plugin/7727?pr=clion) for IntelliJ's CLion. See the wiki
 for [more information](https://wiki.dlang.org/IDEs).
 
+### Prerequisite: DUB
+
+[DUB](https://github.com/dlang/dub) is a build manager for D.
+
+We recommend it as a way to simplify the process of running the tests, but it is still possible to run without DUB if you cannot or do not want to install it.
+
+#### Windows
+
+The [DUB website's download page](https://code.dlang.org/download) has a Windows installer.
+
+#### Mac OS X
+
+* The [DUB website's download page](https://code.dlang.org/download) has OS X binaries.
+* or using [Homebrew](http://brew.sh/): brew install dmd
```

Comment:
nope, this is in the installing DUB section, so the command should say `dub`. Let #59 fix it.